Game/Game/classes/Player.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using Game.interfaces;
+
+namespace Game.classes
+{
+    public abstract class Player : IHuman
+    {
+        public const int MIN_VALUE = 40;
+        public const int MAX_VALUE = 140;
```

Comment:
why did you make them public ? do you need to use them outside of the current hierarchy?

Reply:
Fixed